### DISCUSSION TITLE
First Contribution!

### DISCUSSION THREAD

**Discussion by mariachrisochoou** [ActivityWatch/activitywatch#1037] [Q&A]
Hey, this project is so intersting and I was wondering if there are any good first issues as a first-time contributor that I can contribute to. Any tips? Thank you in advance!

**Comment by m-spangenberg** [comment]
You can filter by 'good first issue' and 'help wanted' labels under any repository by going to the Issues tab.

https://github.com/ActivityWatch/activitywatch/labels/good%20first%20issue
https://github.com/ActivityWatch/activitywatch/labels/help%20wanted
